projects/unore/index.js

```diff
@@ -25,6 +26,18 @@ const ethRiskPools = [
 ]
 const ethSSIPEth = '0x29B4b8674D93b36Bf651d0b86A8e5bE3c378aCF4'
 
+// KAVA pools
+// tokens
+const kavaTokens = [
+  // '0xfA9343C3897324496A05fC75abeD6bAC29f8A40f', // USDC
+]
+// risk pools
+const kavaRiskPools = [
+  '0x6cEC77829F474b56c327655f3281739De112B019'
```

Comment:
is this address correct? I dont see anything there: https://explorer.kava.io/address/0x6cEC77829F474b56c327655f3281739De112B019/transactions

Reply:
Yes, correct. This is USDC pool but we have no any stakes there.